ntbs-service/DataMigration/ImportValidator.cs

```diff
@@ -179,7 +179,7 @@ public ImportValidator(IImportLogger logger, IReferenceDataRepository referenceD
 
             // As we have imported the case manager in a previous step we don't expect to see this message
             var message = "Case manager assigned to notification is not present in NTBS database";
-            validationsResults.Add(new ValidationResult(message, new[] { nameof(details.CaseManagerUsername) }));
+            validationsResults.Add(new ValidationResult(message, new[] { nameof(details.CaseManagerId) }));
```

Comment:
Is this only used while importing a notification? If so, then this check really doesn't make sense anymore. The migration database has provide a username, which we've mapped to an NTBS ID, so we really can be sure that it exists in the NTBS database unless the application has really screwed something up (in which case a validation error wouldn't be appropriate).

Reply:
Agreed, I will change this to just check for case manager status and log if needed